optimum/utils/normalized_config.py

```diff
@@ -211,7 +211,7 @@ class NormalizedConfigManager:
         "blenderbot": BartLikeNormalizedTextConfig,
         "blenderbot_small": BartLikeNormalizedTextConfig,
         "bloom": NormalizedTextConfig.with_args(num_layers="n_layer"),
-        "falcon": NormalizedTextConfig.with_args(num_layers="num_hidden_layers", num_attention_heads="num_kv_heads"),
+        "falcon": NormalizedTextConfig,
```

Comment:
Question: does `NormalizedConfig` have a `NUM_KV_HEADS` attribute to normalize it or not?

Reply:
No